src/med_bench/utils/classifiers.py

```diff
@@ -0,0 +1,423 @@
+from sklearn.linear_model import LogisticRegressionCV, RidgeCV, LassoCV
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.calibration import CalibratedClassifierCV
+import numpy as np
+from numpy.random import default_rng
+from scipy import stats
+import pandas as pd
+from pathlib import Path
+from scipy.stats import bernoulli
+from scipy.special import expit
+
+from itertools import combinations
+from sklearn.model_selection import KFold
+from sklearn.base import clone
+from .utils import get_interactions, _convert_array_to_R
+
+ALPHAS = np.logspace(-5, 5, 8)
+CV_FOLDS = 5
+TINY = 1.e-12
+
+def get_x_classifiers(regularization, forest, calibration, calib_method):
+
+    if regularization:
+        cs = ALPHAS
+    else:
+        cs = [np.inf]
+
+    if not forest:
+        x_clf = LogisticRegressionCV(Cs=cs, cv=CV_FOLDS)
+        xm_clf = LogisticRegressionCV(Cs=cs, cv=CV_FOLDS)
+    else:
+        x_clf = RandomForestClassifier(n_estimators=100,
+                                       min_samples_leaf=10)
```

Comment:
I think that classifiers parameters should be controled by  **kwargs and not hard-coded.